pkg/cmd/server/origin/authenticator.go

```diff
@@ -108,6 +109,18 @@ func newAuthenticator(config configapi.MasterConfig, accessTokenGetter oauthclie
 			group.NewTokenGroupAdder(oauthTokenAuthenticator, []string{bootstrappolicy.AuthenticatedOAuthGroup}))
 	}
 
+	for _, wta := range config.AuthConfig.WebhookTokenAuthenticators {
+		ttl, err := time.ParseDuration(wta.CacheTTL)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Error converting CacheTTL='%s' to duration", wta.CacheTTL)
+		}
+		webhookTokenAuthenticator, err := webhooktoken.New(wta.ConfigFile, ttl)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Failed to create webhook token authenticator for ConfigFile='%s'", wta.ConfigFile)
```

Comment:
Include the error that caused the failure:

`return nil, nil, fmt.Errorf("Failed to create webhook token authenticator for ConfigFile=%q: %v", wta.ConfigFile, err)`

Also replace `'%s'` with `%q` on both messages.